src/uistate.h

```diff
@@ -111,6 +111,35 @@ struct overmap_sidebar_uistate {
     void deserialize( const JsonObject &jo );
 };
 
+// the consume menu reopens when done successfully consuming
+struct consume_menu_uistate {
+    // values[0] = inventory selector selected column index,
+    // values[1] = inventory selector selected index in column
+    std::vector<size_t> consume_menu_selections;
```

Comment:
`size_t` is an issue for de-/serializing (on mac). Use something like `uint64_t` instead.